### DISCUSSION TITLE
Icon Masks

### DISCUSSION THREAD

**Discussion by trollcake** [SFX-WoW/Masque#176] [Issues] [answered]
Using a WA but when i have masque on this comes up 
![image](https://user-images.githubusercontent.com/38572673/39075387-a071863e-44f5-11e8-82d6-3c8696546e02.png)

**Comment by StormFX** [comment]
Looks like you're using a trigger that's causing the buttons to be duplicated. This isn't an issue with Masque.

**Comment by Marving1023** [comment, marked as answer]
If you have ElvUI and AddOnSkins Try /ElvUI =>AddOnSkins => Skins Addon and uncheck WeakAuras Support.

**Comment by StormFX** [comment]
Closing this due to it being an issue with ElvUI.
